src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstBase;
+    private double secondBase;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double firstBase, double secondBase, double height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
+    }
+
+    public double getFirstBase() {
+        return firstBase;
+    }
+
+    public double getSecondBase() {
+        return secondBase;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstBase + secondBase) * height) / 2;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: IsoscelesTrapezoid, area: "
```

Comment:
For enum constants, use the name() method instead of toString() to get their String representation. This ensures that the exact name of the enum constant is returned, even if the toString() method is overridden.